auto_remediate/config_rules.py

```diff
@@ -16,29 +39,59 @@ def rds_instance_public_access_check(self, resource_id):
         Returns:
             boolean -- True if remediation was successful
         """
-        client = boto3.client("rds")
-
-        # unfortunately the resourceId provided by AWS Config is DbiResourceId
-        # and cannot be used in the modify_db_instance function
-        # we therefore need to search all RDS instances
         try:
-            response = client.describe_db_instances()
-
+            response = self.client_rds.describe_db_instances()
+        except:
+            self.logging.error("Could not describe RDS DB Instances.")
+            return False
+        else:
             for instance in response.get("DBInstances"):
```

Comment:
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/rds.html#RDS.Paginator.DescribeDBInstances

Use RDS client's paginators like we did with the user search.

Reply:
#34